.github/workflows/release.yml

```diff
@@ -27,9 +27,9 @@ jobs:
         uses: actions/cache@v2
         with:
           path: vendor/bundle
-          key: ${{ runner.os }}-gem-${{ hashFiles('**/Gemfile.lock') }}
+          key: ${{ runner.os }}-gem-v1-${{ hashFiles('**/Gemfile.lock') }}
           restore-keys: |
-            ${{ runner.os }}-gem-
+            ${{ runner.os }}-gem-v1
       - name: Install Bundle
         run: |
```

Comment:
Did you mean to add in the documentation deployment in the release workflow?

Reply:
I think I know what you mean... the release workflow is already setup to deploy docs, I just needed to update the cache

Reply:
My bad, I saw you removed the docs deploy from the other workflow, didn't see that it was already here!